packages/data/tsconfig.json

```diff
@@ -14,7 +14,5 @@
 		{ "path": "../priority-queue" },
 		{ "path": "../redux-routine" }
 	],
-	"include": [
-		"src/**/*"
-	]
+	"include": [ "src/**/*" ]
```

Comment:
When the latest trunk was merged, unexpected code reformatting was performed.
But since this change was made by the linter, it appears to be OK.